docs/about/cedar-design-system/README.md

```diff
@@ -14,45 +14,71 @@
 
 <cdr-doc-table-of-contents-shell parentSelector='h2' childSelector='h3'>
 
-## About Cedar
-Cedar is REI’s open-source Design System for digital products and experiences. Cedar includes the resources to create user interfaces consistent with REI’s [experience principles](../../foundation/experience-principles/), brand styles, and best practices.
+ A design system is a collection of reusable UI components and guidance to help digital teams make design and development decisions. 
+  
+## Meet Cedar
+Cedar is REI’s open-source design system for digital products and experiences, based on REI’s visual language. Its purpose is simple: to help people create consistent, unified user interfaces. We do this by simplifying common design decisions, freeing folks to focus on groundbreaking work. 
```

Comment:
this will need a space between the heading and the content for the header to render